packages/chain-adapters/src/cosmossdk/thorchain/ThorchainChainAdapter.ts

```diff
@@ -158,7 +158,7 @@ export class ChainAdapter extends CosmosSdkBaseAdapter<KnownChainIds.ThorchainMa
       const msg: Message = {
         type: 'thorchain/MsgDeposit',
         value: {
-          coins: [{ asset: 'rune', amount: bnOrZero(value).toString() }],
+          coins: [{ asset: 'THOR.RUNE', amount: bnOrZero(value).toString() }],
```

Comment:
Does that mean trading into RUNE never worked for KeepKey?

The asset has been set to `rune` since Adam's original implementation in September 2022.

Reply:
@BitHighlander pls ^^
Also relevant: https://discord.com/channels/554694662431178782/886041582422356018/1092531960200581240

Reply:
@0xApotheosis yeah looks like swaps *from* RUNE never worked in the first place and we never spotted it, KK has stronger validation on the `thorchain/MsgDeposit` MsgType:

https://github.com/shapeshift/hdwallet/commit/91d8d40a0c53d26db90189b2ab28e1b1a5d4cfb3#diff-d27caa3e5179cbadb8b52b36dd988e4cc50f0affa4b65e8f6af23a2dd6eae88dR77